Unix/configure

```diff
@@ -464,6 +464,7 @@ else
     configuuid="`echo $USER`"
 fi
 
+
```

Comment:
Sorry, by "Stray?", I meant needless newline inserted for no particular reason. A nit, not important.